src/styles/blocks/cards.scss

```
@@ -0,0 +1,111 @@
+.cards {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  @media screen and (max-width:1200px) {
+    margin: 50px 129px 73px;
+  }
+  .card {
+    width: 200px;
+    background: #fff;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    padding: 32px 16px 0;
+    margin-right: 48px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-top: 50px;
+    }
```

```suggestion
    padding: 32px 16px 0;
```
do not set margins for .card block. instead make .cards__card element (see comment for index.html)
and set margins for it